curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/654212b82fb5cf872f77148f.md

```diff
@@ -0,0 +1,624 @@
+---
+id: 654212b82fb5cf872f77148f
+title: Step 15
+challengeType: 0
+dashedName: step-15
+---
+
+# --description--
+
+Inside the `button` create two more `span` elements. The first span element should have class `playlist-song-artist` and interpolate `song.artist`. The second span element should have class `playlist-song-duration` and interpolate `song.duration`.
+
+# --hints--
+
+Test 1
+
+```js
+
+```
```

Comment:
You should not modify the existing `button` element and its content.

```js
assert.match(code, /<button\s*class=('|")playlist-song-info\1>\s*<span\s*class=\1playlist-song-title\1>\$\{song\.title\}\s*<\/span>\s*.*\s*.*\s*<\/button>/)
```

You should create a `span` element with the class `playlist-song-artist`.

```js
assert.match(code, /<span\s*class=('|")playlist-song-artist\1>/)
```

Your `span` element with the class `playlist-song-artist` should interpolate `song.artist` as its content.

```js
assert.match(code, /<span\s*class=('|")playlist-song-artist\1>\s*\$\{song\.artist\}\s*<\/span>/)
```

You should create a `span` element with the class `playlist-song-duration`.

```js
assert.match(code, /<span\s*class=('|")playlist-song-duration\1>\s*/)
```

Your `span` element with the class `playlist-song-duration` should interpolate `song.duration` as its content.

```js
assert.match(code, /<span\s*class=('|")playlist-song-duration\1>\s*\$\{song\.duration\}<\/span>/)
```